README.md

```diff
@@ -211,8 +308,8 @@ Now try:
 $ ls
 ```
 
-You should see the file you just created, `hello_world.rb`, in the working directory. Note that this is an
-empty file and has nothing inside of it, because you just created it.
+You should see the file you just created, `hello_world.rb`, in the working directory.
+Note that this is an empty file and has nothing inside of it, because you just created it.
 
 ### Making New Directiries Using `mkdir`
```

Comment:
typo. I've now corrected two typos in this text. Is the spellcheck in your editor catching these?